### PR TITLE
Final fix to get campus and ctown cronjobs working

### PR DESCRIPTION
**Pull request by meganle**
## Overview
Last one I promise (hopefully 🤞). This should finally, actually get cronjobs running inside of Docker containers.

## Changes Made
- Moved python script inside actual python files
- Did some weird stuff in `update_db.txt` to run the scripts and capture the output to `/var/log/cron.log`

## Test Coverage
Tested by running the Docker container locally and making sure the logs looked correct by setting cronjobs to run every minute

## Next Steps (delete if not applicable)
Deploy and watch containers carefully

### CODE REVIEW

File: update_db.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-0 9,15,21 * * * ./update_campus.sh
-@weekly ./update_ctown.sh
+0 9,15,21 * * * cd /usr/src/app; /usr/local/bin/python update_campus.py >> /var/log/cron.log 2>&1
+@weekly cd /usr/src/app; /usr/local/bin/python update_ctown.py >> /var/log/cron.log 2>&1
```
Comment (leewang-codes) [inline update_db.txt]: what is `2>&1` for?

**Review by leewang-codes — APPROVED** [review]
LGTM

File: update_db.txt (hunk 1 above)
Comment (meganle) [reply update_db.txt]: This allows error messages to also be logged cause before it was suppressed. https://stackoverflow.com/questions/818255/in-the-shell-what-does-21-mean

Comment (yuna878) [inline update_db.txt]: Change `@weekly` to `0 0 * * 0` (=every Sunday midnight) please

**Review by yuna878 — APPROVED** [review]
💯

File: update_db.txt (hunk 1 above)
Comment (meganle) [reply update_db.txt]: oh wait we should adjust it for utc time like we do for the update campus job

Comment (yuna878) [reply update_db.txt]: `0 5 * * 0` = 5AM UTC / 12AM EST Sundays

**Pull request merged**